src/components/common/UploadDocumentEwallet.tsx

```
@@ -1,108 +1,558 @@
+			} else {
+				parsedJson = vc.json;
+			}
+			const documentName = parsedJson?.credentialSchema?.title ?? 'Unknown document';
+			let errorMessage;
+			if (docError instanceof Error && docError.message.includes('does not match any of the accepted document types')) {
+				errorMessage = 'Document type not accepted';
+			} else if (
+				typeof docError === 'object' &&
+				docError !== null &&
+				'response' in docError &&
+				typeof (docError as any).response === 'object' &&
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)